mcore-mode.el

```diff
@@ -73,22 +73,22 @@
        ;; Inline comment “-- ...”
        (modify-syntax-entry ?/ ". 12a" synTable)
        (modify-syntax-entry ?- "_ 123" synTable)
+       (modify-syntax-entry ?' "\"" synTable)
        (modify-syntax-entry ?\n ">" synTable)
        synTable))
 
 ;;;;;;;;;;;;;;
 ;; prettify ;;
 ;;;;;;;;;;;;;;
 
-(defvar mcore-prettify-symbols-alist
-  '(("lam" . ?λ))
-  "List of syntax to prettify for `mcore-mode'.")
+(defun prettify-mcore-list ()
+  (setq prettify-symbols-alist
+        '(
+          ("lam" . 955) ;λ
+          )))
+
+(add-hook 'mcore-mode-hook 'prettify-mcore-list)
 
-(if (boundp 'prettify-symbols-alist)
```

Comment:
I'm sceptical of removing this check. We could have someone with a minor mode using the `prettify-symbols-alist` (I know people who use it for all `prog-mode` derivatives). The new code above overwrites any existing list to only use what we deem useful for `mcore-mode`. Or am I missing something?

Reply:
(on second thought, it's not removing the check that is the problem, but replacing `push` by `setq`)

Reply:
Do we want to keep the other user defined prettifiers? Couldn't they maybe conflict with some keywords that we have in mcore?

Reply:
I think there are some common symbolic ones that some people use, for example replacing `->` by `→`, `not` by `¬`, etc.

Reply:
I think the check is needed, since `prettify-symbols-alist` is not defined for older versions of emacs.